src/dashboard/infrastructure/integrations/site-kit.php

```diff
@@ -109,6 +120,24 @@ public function is_onboarded() {
 		return ( $this->is_site_kit_installed() && $this->is_setup_completed() && $this->is_connected() );
 	}
 
+	/**
+	 * Can user read site kit data.
+	 *
+	 * @param string $slug The slug of the data.
```

Comment:
Slug seems related to URLs, at least in my mind. Perhaps rename to `key`, or `name`?